recipe_scrapers/_utils.py

```diff
@@ -202,17 +221,15 @@ def url_path_to_dict(path):
         r"^"
         r"((?P<schema>.+?)://)?"
         r"((?P<user>.+?)(:(?P<password>.*?))?@)?"
-        r"(?P<host>.*?)"
```

Comment:
+1 for this host-matching fixup - let's add a test case for it though too, while we're here, to prevent unintended regressions.